teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -30,10 +35,18 @@
       converter = GraffitiConverter.class,
       paramLabel = "<GRAFFITI STRING>",
       description =
-          "Graffiti to include during block creation (gets converted to bytes and padded to Bytes32).",
+          "Graffiti value to include during block creation. Value gets converted to bytes and padded to Bytes32).",
       arity = "1")
   private Bytes32 graffiti;
 
+  @Option(
+      names = {"--validators-graffiti-file"},
+      paramLabel = "<GRAFFITI FILE>",
+      description =
+          "File to load graffiti value to include during block creation. Value gets converted to bytes and padded to Bytes32).",
```

Comment:
(nit) Might be worth noting here that if the file read fails, we'll fall back to the `--graffiti` value if supplied:
```suggestion
          "File to load graffiti value to include during block creation. Value gets converted to bytes and padded to Bytes32).  If file reading fails during block creation, teku will fall back to any value supplied via --graffiti.",
```